### PR TITLE
fix(web): default kbd ui name is now generic

### LINKED ISSUE #99: bug(web): no ability to set name of the default language
No matter the actual language or keyboard installed on a system, Keyman labels the default keyboard "English."  Shouldn't we perform an actual check somewhere to detect the language and run with that?  What if someone wishes to use an alternate name for it?

### PR DESCRIPTION
**Pull request by jahorton**
Fixes #99.

Note that this only affects the desktop UIs, as mobile form factors currently require an active keyboard to be present.

### CODE REVIEW

File: web/source/kmwuifloat.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ if(!window['keyman']['ui']['name']) {
       ui.innerDiv.appendChild(ui.KeyboardSelector);  //this may need to be moved up.... 
 
       // Check required interface alignment and default keyboard
-      var opt=util['getOption']('ui'),dfltKeyboard='English';
+      var opt=util['getOption']('ui'),dfltKeyboard='(System keyboard)';
```
Comment (darcywong00) [inline web/source/kmwuifloat.ts:129]: Can `value` in l. 240 get removed too?
https://github.com/keymanapp/keyman/blob/0093c8b991d402bc34f2ed54da6a30b3c1d8a16c/web/source/kmwuifloat.ts#L240

**Comment by darcywong00** [comment]
Since the issue title is 
> no ability to set name of the default language

Confirming the fix for the issue is renaming the default keyboard to "System Keyboard" instead of KMW  checking the actual system keyboard and running with it

**Comment by jahorton** [comment]
> Since the issue title is
> 
> > no ability to set name of the default language
> 
> Confirming the fix for the issue is renaming the default keyboard to "System Keyboard" instead of KMW checking the actual system keyboard and running with it

https://github.com/keymanapp/keyman/issues/99#issuecomment-730096887

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.225-beta](https://keyman.com/downloads/releases/beta/14.0.225)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.9-alpha](https://keyman.com/downloads/releases/alpha/15.0.9)

**Pull request merged**